src/pages/Lend/LoanChannelCategoryPage.vue

```diff
@@ -372,130 +372,261 @@ export default {
 		},
 		getMetaInfo() {
 			switch (this.targetedLoanChannel) {
+				case '1-billion-in-change':
```

Comment:
I want us to use this ticket as an opportunity to set the meta title based on the category name from the API, so that we don't have to hard-code it in this file any more. The meta title just needs to be `${category.name} | Invest & Support`.

The category name is already fetched from the API in the `LoanChannelCategoryClimateExperiment` and `LoanChannelCategoryControl` components using functions from the `loanChannelUtils` file, so I would recommend defining the meta info in those components instead of this one. I would also recommend putting the function(s) that get/set the meta info in the `loanChannelUtils` file so that it can be shared by both category layout components.